docs/fundamentals-and-concepts/networks.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Networks
+sidebar_position: 30
+---
+
+Stellar has three networks: the public network (Mainnet, also called Pubnet or the Public Network), the test network (Testnet), and a dev network (Futurenet). Mainnet is the main network used by applications in production. It connects to real financial rails and requires XLM to cover minimum balances, transaction fees, and rent. The Testnet is a smaller, free-to-use network maintained by SDF that functions like the Mainnet but doesn’t connect to real money. It has a built-in testnet XLM faucet (called Friendbot), and it's the best place for developers to test their applications. Futurenet is a dev network you can use to test more bleeding edge features that also has access to its own Friendbot.
+
+## Stats: Mainnet versus Testnet versus Futurenet
+
+### Mainnet
+
+- Validator nodes are run by the public
+- SDF offers a free [Horizon instance](https://horizon.stellar.org/) to interact with the Mainnet with a limited set of history, or you can [run your own](https://developers.stellar.org/docs/run-api-server)
+- You need to fund your account with XLM from another account
+- Mainnet is limited to 1,000 operations per ledger and will be limited to a maximum of 30 smart contract transactions per ledger (the precise amount of smart contract txs per ledger can vary greatly depending on transaction [resource limits](https://soroban.stellar.org/docs/fundamentals-and-concepts/fees-and-metering#resource-limits))
+  - See more detailed smart contract network settings in the section on [Fees and Metering](https://soroban.stellar.org/docs/fundamentals-and-concepts/fees-and-metering) in the Soroban docs
+- No publicly available RPC, see RPC service providers [here](https://soroban.stellar.org/docs/reference/rpc-list)
+
+### Testnet
+
+- SDF runs three core validator nodes
+- SDF offers a free [Horizon instance](https://horizon-testnet.stellar.org/) you can use to interact with the Testnet
+- Friendbot is a faucet you can use for free Testnet XLM
+- Testnet is limited to 100 operations per ledger and one smart contract transaction per ledger
```

Comment:
Is one smart contract transaction per ledger accurate?  Just want to verify that's the case before we enshrine it in the docs.